torchao/quantization/quant_primitives.py

```diff
@@ -692,6 +714,56 @@ def _choose_qparams_affine(
     return scale.to(dtype=scale_dtype), zero_point.to(dtype=zero_point_dtype)
 
 
+# Float8
+def quantize_affine_float8(
+    input: torch.Tensor,
+    block_size: Tuple[int, ...],
+    output_dtype: torch.dtype,
+    quant_min: Optional[Union[int, float]] = None,
+    quant_max: Optional[Union[int, float]] = None,
+    zero_point_domain: ZeroPointDomain = ZeroPointDomain.INT,
+    reduce_amax: bool = False,
+) -> Float8Tensor:
+    return hp_tensor_to_float8_dynamic(
```

Comment:
I thought adding float8 to AQT would mean the casts in AQT would be used, it's a bit confusing to use AQT but also use external utilies for casting.

Reply:
The final goal is for the AQT cast to be used. I was facing some errors while trying to use AQT's cast, was trying to resolve those.